communication/contributor-comms/blogging-resources/sig-spotlights.md

```diff
@@ -14,4 +14,7 @@ These are all the spot lights that are published to date:
 | [Network](https://github.com/kubernetes/community/blob/master/sig-network/README.md) | [network](https://github.com/kubernetes/kubernetes/labels/sig%2Fnetwork) | [Spotlight on SIG Network](https://www.kubernetes.dev/blog/2023/05/09/sig-network-spotlight-2023/) | Sujay Dey | 2023-05-09 |
 | [CLI](https://github.com/kubernetes/community/blob/master/sig-cli/README.md) | [CLI](https://github.com/kubernetes/kubernetes/labels/sig%2Fcli) | [Spotlight on SIG CLI](https://www.kubernetes.dev/blog/2023/07/20/sig-cli-spotlight-2023/) | Arpit Agrawal | 2023-07-20 |
 | [Contributor Experience](https://github.com/kubernetes/community/blob/master/sig-contributor-experience/README.md) | [contributor-experience](https://github.com/kubernetes/kubernetes/labels/sig%2Fcontributor-experience) | [Spotlight on SIG Contributor Experience](https://kubernetes.io/blog/2023/08/14/sig-contribex-spotlight-2023/) | Fyka Ansari | 2023-08-14 |
+| [Architecture](https://github.com/kubernetes/community/blob/master/sig-architecture/README.md) | [architecture](https://github.com/kubernetes/kubernetes/labels/sig%2Farchitecture) | [SIG Architecture: Production Readiness Spotlight](https://kubernetes.io/blog/2023/11/02/sig-architecture-production-readiness-spotlight-2023/) | Frederico Muñoz | 2023-11-02 |
+| [Architecture](https://github.com/kubernetes/community/blob/master/sig-architecture/README.md) | [architecture](https://github.com/kubernetes/kubernetes/labels/sig%2Farchitecture) | [SIG Architecture: Conformance Spotlight](https://kubernetes.io/blog/2023/10/05/sig-architecture-conformance-spotlight-2023/) | Frederico Muñoz | 2023-10-05 |
 | [Testing](https://github.com/kubernetes/community/tree/master/sig-testing#readme) | [testing](https://github.com/kubernetes/kubernetes/labels/sig%2Ftesting) | [Spotlight on SIG Testing](https://kubernetes.io/blog/2023/11/24/sig-testing-spotlight-2023/) | [Sandipan Panda](https://github.com/sandipanpanda) | 2023-11-24 |
+| [Release](https://github.com/kubernetes/community/tree/master/sig-release#readme) | [release](https://github.com/kubernetes/kubernetes/labels/sig%2Frelease) | [Spotlight on SIG Testing](https://kubernetes.io/blog/2023/11/24/sig-testing-spotlight-2023/) | Nitish Kumar | 2024-01-15 |
```

Comment:
Update this with the spotlight on SIG Release https://kubernetes.io/blog/2024/01/15/sig-release-spotlight-2023/

Reply:
Yes I have made the changes